rollup.config.js

```diff
@@ -6,28 +6,36 @@ import commonjs from 'rollup-plugin-commonjs';
 import { terser } from 'rollup-plugin-terser';
 import serve from 'rollup-plugin-serve';
 
+const langFiles = ['en-GB', 'es-ES', 'it-IT', 'pt-PT', 'zh-CN'];
```

Comment:
Hi @pedroladeira thanks for your contribution. 

But since HelloWeek can support more languages in the future and to avoid having to change `rollup.config.js` every time we add a new language, I think you could simplify this array, using fs.readdirSync to list all files in the src/langs folder.

Reply:
Thanks. I will do the change